src/test/java/com/iexec/worker/tee/pre/PreComputeArgsTests.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.worker.tee.pre;
+
+import com.iexec.common.replicate.ReplicateStatusCause;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static com.iexec.common.precompute.PreComputeUtils.*;
+import static com.iexec.common.utils.IexecEnvUtils.*;
+
+class PreComputeArgsTests {
```

Comment:
I don't see the case where an unknown cause is passed to `buildReplicateCauseIfMissing` which should return `null` if I understand correctly.

Reply:
I agree, updated thanks!

Reply:
This is just above code coverage improvement in tests

Reply:
Totally agree :)

Reply:
Thanks for the update

Reply:
Thanks for the update